gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/TableRow.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.bigquery;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.api.client.util.Lists;
+import com.google.common.base.Function;
+import com.google.common.collect.ImmutableList;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Google BigQuery Table Row class. Objects of this class contain a list of {@link FieldValue}, one
+ * for each field in the table.
+ */
+public class TableRow implements Serializable {
```

Comment:
I am not totally against but I see no reason (or a potential benefit) for making users go an extra step (and call `values()`) for getting the values. Do you?